Symfony/CS/ConfigInterface.php

```diff
@@ -49,9 +51,18 @@ public function getFinder();
     public function getLevel();
 
     /**
-     * Returns the fixers to run.
+     * Set the fixers to be used.
+     *
+     * @param string[] $fixers
+     *
+     * @return ConfigInterface
+     */
+    public function fixers(array $fixers);
+
+    /**
+     * Returns the fixers.
      *
-     * @return array A list of fixer names
+     * @return FixerInterface[] A list of fixers
```

Comment:
yes, I spotted only way place this is the case (didn't check the test yet)

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Console/ConfigurationResolver.php#L264